ServerCore/Pages/Teams/TeamCompositionSummary.cshtml.cs

```diff
@@ -110,22 +110,22 @@ public IActionResult OnPost(SortEnum sortBy, SortDirectionEnum sortDirection)
                 });
         }
 
-        private Func<TeamComposition, string> GetSortSelector()
+        private Func<TeamComposition, object> GetSortSelector()
```

Comment:
uh are we able to typecheck here? like, Sortable? or something?

or should we do a `ToInt` or `ToNum` or something in the `case`s?